cpp/ql/test/library-tests/dataflow/modelgenerator/dataflow/summaries.cpp

```diff
@@ -0,0 +1,130 @@
+using size_t = decltype(sizeof(int));
+
+size_t strlen(const char* str);
+char* strcpy(char* dest, const char* src);
+
+namespace Models {
+    struct BasicFlow {
+        int* tainted;
+
+        //No model as destructors are excluded from model generation.
+        ~BasicFlow() = default;
```

Comment:
Maybe consider adding more testcases for members where we expect not to get any models. It is a good idea (as you have already done) to narrow the number of produced models to only "effectively public" endpoints as loading data extensions can be somewhat expensive (if there are 10k's+ models).

Reply:
7a56266c76753aacbe19843c5685f70f8703806a adds a bunch of tests. I admit the number of tests was a bit on the light side 😂